### ISSUE TITLE
Webdriver manager problem while running the framework in docker container

### ISSUE THREAD

**Issue by baradhiren** [jdi-testing/jdi-light#3285]
I am trying to run my jdi-light-testng template with Docker container. I am downloading the latest chrome to run the tests headlessly. But when I do so I am getting following error:

```37:44.495 [JDI:INFO]: [18] Failed to download driver (CHROME latest) of latest version:TRY TO GET DRIVER PREVIOUS VERSION                                                                                                                    37:44.499 [io.github.bonigarcia.wdm.WebDriverManager:INFO]: Reading https://chromedriver.storage.googleapis.com/ to seek chromedriver 
37:45.485 [io.github.bonigarcia.wdm.online.HttpClient:ERROR]: Error HTTP 404 executing https://chromedriver.storage.googleapis.com/LATEST_RELEASE_92.0.4515.107
37:45.486 [io.github.bonigarcia.wdm.versions.VersionDetector:WARN]: Exception reading https://chromedriver.storage.googleapis.com/LATEST_RELEASE_92.0.4515.107 to get latest version of chromedriver (Error HTTP 404 executing https://chromedriver.storage.googleapis.com/LATEST_RELEASE_92.0.4515.107)
37:45.788 [io.github.bonigarcia.wdm.WebDriverManager:INFO]: Exporting webdriver.chrome.driver as /root/.cache/selenium/chromedriver/linux64/92.0.4515.107/chromedriver 
37:45.789 [JDI:INFO]: [18] Download driver: 'CHROME X64 92.0.4515.107' successfully  
Starting ChromeDriver 92.0.4515.107 (87a818b10553a07434ea9e2b6dccf3cbe7895134-refs/branch-heads/4515@{#1634}) on port 31691 Only local connections are allowed. 
Please see https://chromedriver.chromium.org/security-considerations for suggestions on keeping ChromeDriver safe. [1631039865.795][SEVERE]: bind() failed: Cannot assign requested address ChromeDriver w(99)  as started successfully. 
37:46.267 [io.github.bonigarcia.wdm.WebDriverManager:INFO]: Using chromedriver 93.0.4577.15 (resolved driver for Chrome 93)
37:46.268 [io.github.bonigarcia.wdm.WebDriverManager:INFO]: Exporting webdriver.chrome.driver as /root/.cache/selenium/chromedriver/linux64/93.0.4577.15/chromedriver 
37:46.269 [JDI:INFO]: [18] Download driver: 'CHROME X64' successfully```

I am not sure what I am missing here.

**Comment by baradhiren** [comment]
I have seen this error: Error HTTP 404 executing https://chromedriver.storage.googleapis.com/LATEST_RELEASE_92.0.4515.107 in Webdriver manager repo as well but it was fixed in WDM 4.3.1. (Which is updated in [this commit](https://github.com/jdi-testing/jdi-light/blob/09a3d69321c2794f66c75038d3f494885c89842c/jdi-light/pom.xml)).

**Comment by pnatashap** [comment]
WDM was updated
